### ISSUE TITLE
www.datasciencetoolkit.org is down

### ISSUE THREAD

**Issue by mlucool** [petewarden/dstk#61]
Not sure if this is the right place to post this, but www.datasciencetoolkit.org/ is down. Verified by: http://www.downforeveryoneorjustme.com/www.datasciencetoolkit.org/

**Comment by mikedolanfliss** [comment]
I experienced the same.  I found a few forks my R code could reference... but then ended up with using the new geocoding API from the census.  Note Pete Warden's blog post on DSTK and "new management."  I'm not so sure they're maintaining it... which is too bad.  It's a beautiful package.

http://petewarden.com/2015/09/27/openheatmap-and-datasciencetoolkit-under-new-management/

**Comment by fredclown** [comment]
It is truly a bummer as dstk is the only solution I have found for offline geocoding. While the census bureau does have an online geocoder there are businesses that could never use it (such a US medical institutions) because they would have to transmit the address data over the internet.  DSTK scratched this itch right nicely. Too bad the latest TIGER data in it is from 2012 I believe.

**Comment by mikedolanfliss** [comment]
Yeah.  FWIW, this thread seems to celebrate GisGraphy.  Seems to work
locally if you can manage to set it up.

http://gis.stackexchange.com/questions/18871/is-there-an-open-source-geocoding-tool-which-can-be-used-commercially

On Fri, Jan 22, 2016 at 12:15 PM, fredclown notifications@github.com
wrote:

> It is truly a bummer as dstk is the only solution I have found for offline
> geocoding. While the census bureau does have an online geocoder there are
> businesses that could never use it (such a US medical institutions) because
> they would have to transmit the address data over the internet. DSTK
> scratched this itch right nicely. Too bad the latest TIGER data in it is
> from 2012 I believe.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/petewarden/dstk/issues/61#issuecomment-173981852.

## 

---

Mike Dolan Fliss, MSW
mike.dolan.fliss@gmail.com
UNC-CH Epidemiology PhD student

## NC public health advocate!

"We work on ourselves in order to help others,
but also we help others in order to work on ourselves."
- Pema Chodron

“Upon this gifted age, in its dark hour
Rains from the sky a meteoric shower
Of facts…they lie, unquestioned, uncombined.
Wisdom enough to leach us of our ill
Is daily spun; but there exists no loom
To weave it into a fabric.”
- Edna St. Vincent Millay, 1939

There are two kinds of people in the world:
Those who can extrapolate from incomplete data.
